packages/protocol/thread/thread.ts

```diff
@@ -600,6 +601,38 @@ class _ThreadFront {
     return rv;
   }
 
+  // Same as evaluate, but returns the result without wrapping a ValueFront around them.
+  // TODO Replace usages of evaluate with this.
+  async evaluateNew({
+    asyncIndex,
+    text,
+    frameId,
+    pure = false,
+  }: {
```

Comment:
Note @markerikson This method will probably be helpful for you in the "kill the fronts" effort.